modules/woocommerce-analytics/classes/wp-woocommerce-analytics-universal.php

```diff
@@ -33,14 +33,15 @@ public function __construct() {
 
 		// Capture cart events
 		add_action( 'woocommerce_add_to_cart', array( $this, 'capture_add_to_cart' ), 10, 6 );
+		add_action( 'woocommerce_bundled_item_before_add_to_cart', array( $this, 'capture_bundle_add_to_cart' ), 10, 6 );
 
 		// single product page view
 		add_action( 'woocommerce_after_single_product', array( $this, 'capture_product_view' ) );
 
 		add_action( 'woocommerce_after_cart', array( $this, 'remove_from_cart' ) );
 		add_action( 'woocommerce_after_mini_cart', array( $this, 'remove_from_cart' ) );
 		add_action( 'wcct_before_cart_widget', array( $this, 'remove_from_cart' ) );
-		add_filter( 'woocommerce_cart_item_remove_link', array( $this, 'remove_from_cart_attributes' ), 10, 2 );
```

Comment:
Curious here why this change?

Reply:
That is an oversight on my part. I was testing `woocommerce_cart_item_remove` and forgot to change it back before committing. Thanks.